cmd/context/context.go

```diff
@@ -0,0 +1,136 @@
+// Copyright 2021 The Okteto Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package context
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/okteto/okteto/cmd/utils"
+	"github.com/okteto/okteto/pkg/analytics"
+	okContext "github.com/okteto/okteto/pkg/cmd/context"
+	k8Client "github.com/okteto/okteto/pkg/k8s/client"
+	"github.com/okteto/okteto/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+type ContextOptions struct {
+	Token           string
+	isOktetoCluster bool
+}
+
+// Context points okteto to a cluster.
+func Context() *cobra.Command {
+	ctxOptions := &ContextOptions{}
+	cmd := &cobra.Command{
+		Use:     "context [url|k8s-context]",
+		Aliases: []string{"ctx"},
+		Args:    utils.MaximumNArgsAccepted(1, "https://okteto.com/docs/reference/cli/#context"),
+		Short:   "Points okteto to a cluster",
+		Long: `Points okteto to a cluster
+
+Run
+    $ okteto context
+
+and this command will ask you for the cluster you want to operate in okteto.
```

Comment:
what do we mean by "the cluster you want to operate"?  This sounds more like a "cluster-management" operation.  Also, this is not asking for the cluster, but an existing kubernetes context isn't it?

Reply:
As we mention in the spec, it "points okteto to a cluster". Whether is Okteto Cloud, an instance of Okteto Enterprise, or **maybe** a Kubernetes context/namespace defined in the local `.kubeconfig` file

Reply:
@rlamana could you link to the spec?

Reply:
My comment here is on the text we display. Asking for "a cluster" is different from asking from "a context name" vs "a URL to the okteto instance". If someone asks me for a cluster, my first reaction would be to put the cluster name, or maybe the API. 

Given that this is a multi-shape field (it can be a context name or the URL of an okteto instance), I think we need to be very careful of the terms we use to avoid confusion.

Reply:
Here: https://github.com/okteto/wiki/blob/main/specs/okteto-dev.md#context